external-builds/pytorch/repo_management.py

```diff
@@ -261,6 +261,22 @@ def do_checkout(args: argparse.Namespace, custom_hipify=do_hipify):
     except subprocess.CalledProcessError:
         print("Failed to fetch git submodules")
         sys.exit(1)
+    # Clean up unwanted directories
+    exclude_paths = [
```

Comment:
Let's add more context to this comment. I'm also not thrilled with pytorch-specific paths being in this file that is shared with torchaudio, torchvision, and triton, but that's fine.
```suggestion
    # Delete directories which are the source of flaky pytorch 
    # checkouts on Windows and are not used during the build.
    # See https://github.com/ROCm/TheRock/issues/1149.
    exclude_paths = [
```